addons/account_edi_ubl_cii/models/account_move_send.py

```diff
@@ -189,6 +189,12 @@ def _hook_invoice_document_after_pdf_report_render(self, invoice, invoice_data):
         reader_buffer.close()
         writer_buffer.close()
 
+    def _additional_document_reference_get_custom_nodes(self, invoice):
```

Comment:
Not important, but I don't really like the name of this function (is the function a getter? and am I getting a custom node for an additional document reference? from it?)